team.html

```diff
@@ -354,7 +354,7 @@ <h1 class="clickable">The OTC Team</h1>
 					>
 						<img
 							src="static/img/globe.svg"
-							alt="Harsh Kapadia's website: https://pranavdani.github.io"
+							alt="Pranav Dani's website: https://pranavdani.github.io"
```

Comment:
Harsh yaar

Reply:
Oops. My bad. I'll correct this.

Reply:
Oh you've already corrected it. Thank you. My bad.